examples/se_2_3_localization.cpp

```diff
@@ -46,26 +49,73 @@
  * Note that the linear velocity v is expressed in a frame whose origin coincides with
  * the robot frame but has a orientation which is same as the world frame
  *
- *  The discrete-time system propagation can be written as,
- *   p_{k+1} = p_{k} + R_{k} (R_{k}.T v_{k} dt + 0.5*dt*dt*acc_{k})
- *   R_{k+1} = R_{k} Exp_SO3(omega_{k} dt + w_omega)
- *   v_{k+1} = v_{k} + R_{k} (acc_{k} dt + w_acc)
- *
- * Note that these equations are modified to obey the group composition rules.
- *
- * where,
- *        - R_{k}.T is the transpose of R_{k}
- *        - acc_{k} = alpha_{k} + R_{k}.T g
- *        - w_omega is the additive white noise affecting the gyroscope measurements
- *        - w_acc is the additive white noise affecting the linear accelerometer measurements
- *
- *    Q = diagonal(0_{3x1}, 0_{3x1}, 0_{3x1}, sigma_omegax^2, sigma_omegay^2, sigma_omegaz^2, sigma_accx^2, sigma_accy^2, sigma_accz^2).
- *
- *  The exogenous input u becomes,
- *   u = (R_{k}.T v dt + 0.5*dt*dt*acc_{k}, omega_{k} dt, acc_{k} dt)
- *
- * Note that all the elements of the input vector u belong in the IMU frame/robot frame.
- *
+ * Consider robot coordinate frame B and world coordinate frame A.
+ * - p is the position of the origin of the robot frame B with respect to the world frame A
+ * - R is the orientation of the robot frame B with respect to the world frame A
+ * - v is the velocity of the robot frame with respect to the world frame,
+ *            expressed in a frame whose origin coincides with the robot frame, oriented similar to the world frame
+ *            (it is equivalent to p_dot in continuous time. This is usually called mixed-frame representation
+ *            and is denoted as (B[A] v_AB), where B[A] is the mixed frame as described above.
+ *            For reference, please see "Multibody Dynamics Notation" by Silvio Traversaro and Alessandro Saccon.
+ *            Link: https://research.tue.nl/en/publications/multibody-dynamics-notation-version-2)
+ * - a is the frame acceleration in mixed-representation (equivalent to p_doubledot in continuous time).
+ * - omega_b as the angular velocity of the robot expressed in the robot frame
+ *
+ * The kinematic equations (1) can be written as,
+ * p <-- p + v dt + 0.5 a dt^2
+ * R <-- R Exp_SO3(omega_b)
+ * v <-- v + a dt
+ *
+ * We would like to express these equations in the form,
+ * X <-- X* Exp(u)
```

Comment:
Use space left and right of `*`

Reply:
Fixed in https://github.com/artivis/manif/pull/154/commits/ded121b75f0f401c518336cf0d31ad862be50275